src/Gui/Viewer/CameraManipulator.hpp

```diff
@@ -123,6 +123,17 @@ class RA_GUI_API CameraManipulator : public QObject
   signals:
 
   protected:
+    std::tuple<Scalar, Scalar> computeDeltaMouseMove( const QMouseEvent* mouseEvent ) {
```

Comment:
Why using a std::tuple instead of a std::pair here ?

Reply:
right, I swith to pair.